test/single-sample/GATKSVPipelineSingleSampleTest.test_na19240.json

```diff
@@ -192,6 +192,12 @@
   "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.run_vcf_qc" : false,
   "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.max_ref_panel_carrier_freq": 0.03,
 
+  "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.protein_coding_gtf" : "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/gencode.canonical_pc.gtf.gz",
+  "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.linc_rna_gtf" :       "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/gencode.lincRNA.gtf.gz",
+  "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.promoter_bed" :       "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/promoter.bed",
+  "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.noncoding_bed" :       "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/noncoding.sort.hg38.bed",
+  "GATKSVPipelineSingleSampleTest.GATKSVPipelineSingleSample.annotation_sv_per_shard" : "5000",
+
```

Comment:
Can you add this to the json in `/inputs` as well?

Reply:
These inputs are in `inputs/GATKSVPipelineSingleSample.ref_panel_1kg.na12878.json` -- am I missing somewhere else?

Reply:
Sorry I missed that. All is well!